cmd/descheduler/app/options/options.go

```diff
@@ -55,4 +55,6 @@ func (rs *DeschedulerServer) AddFlags(fs *pflag.FlagSet) {
 	fs.BoolVar(&rs.DryRun, "dry-run", rs.DryRun, "execute descheduler in dry run mode.")
 	// node-selector query causes descheduler to run only on nodes that matches the node labels in the query
 	fs.StringVar(&rs.NodeSelector, "node-selector", rs.NodeSelector, "Selector (label query) to filter on, supports '=', '==', and '!='.(e.g. -l key1=value1,key2=value2)")
+	// max-no-pods-to-evict limits the maximum number of pods to be evicted per node by descheduler.
+	fs.IntVar(&rs.MaxNoOfPodsToEvictPerNode, "max-no-pods-to-evict", rs.MaxNoOfPodsToEvictPerNode, "Limits the maximum number of pods to be evicted per node by descheduler")
```

Comment:
what about MaxPodsToEvictPerNode?

Reply:
Given it is a variable name and not the flag/option name, we can change it later to make the name more clear. Though, `MaxNoOfPodsToEvictPerNode` ~ `MaxPodsToEvictPerNode`.

Reply:
That's fine. But now i think to make the flag : max-pods-to-evict-per-node to make it clear in the flag itself that its per node

Reply:
Done